package/MDAnalysis/core/universe.py

```diff
@@ -1436,6 +1443,27 @@ def from_smiles(cls, smiles, sanitize=True, addHs=True,
 
         return cls(mol, **kwargs)
 
+    def guess_TopologyAttr(self, context, to_guess):
+        """guess attributes passed to the universe within specific context
+
+        Parameters
+        ----------
+        context: string or Guesser class
```

Comment:
Add a sentence for each parameter that also explains what it does — see other docs